packages/ckeditor5-html-support/tests/htmlcomment.js

```diff
@@ -0,0 +1,184 @@
+/**
```

Comment:
I'd add some tests with comments at the edge of the content (one for upcast, downcast, and post-fixer - even if this does not work currently as expected).

Reply:
Done.
After adding new tests, one is failing currently.